data/campaigns/Delfadors_Memoirs/scenarios/07_Night_in_the_Swamp.cfg

```diff
@@ -130,7 +130,7 @@
         [if]
             [variable]
                 name=generators_destroyed
-                greater_than={ON_DIFFICULTY 1 2 2}
+                equals={ON_DIFFICULTY 2 2 3}
```

Comment:
Is this correct? I'm no expert with WML but it looks to me like it's changing from:
Easy: > 1
Normal: > 2
Hard: > 2

to:
Easy: == 2
Normal: == 2
Hard: == 3

I suppose `generators_destroyed` is a variable you've introduced with the PR as a whole, so I'm guessing this means your original implementation for Normal was incorrect, since the number of generators is 2, 2, 3 for the respective difficulties.

Reply:
> this means your original implementation for Normal was incorrect.
That statement is correct, there are 2 generators on EASY and NORMAL and 3 on HARD and while testing the commit 362ebc0 I found that the game was not finishing on NORMAL so I decided to fix it and change greater_than to equals as that made more sense imo.

In conclusion, the lines erased in 362ebc0 from generators.cfg are not the exact same as the ones I added to the scenario.cfg so I added a bug, which then I had to patch up on 5eaba0f.

Reply:
Was just worried at first that you were changing something but then afterwards noticed the bug for Normal. In general (from a programming perspective) `>` or `>=` is safer practice than just testing for equality but I'm okay with leaving this PR as using `equals`.

I'll try to retest before merging, once the string freeze is lifted, but if it's largely the same (functionally) as what you originally submitted a few weeks ago that should be good.